src/app/page.tsx

```diff
@@ -7,28 +7,37 @@ import { Button } from "@/components/ui/button";
 import MenuTheme from "@/components/menuTheme/menuTheme";
 import Header from "@/components/header/Header";
 import { GoogleAnalytics } from "@next/third-parties/google";
+import { LanguageContext } from "../context/LanguageContext";
+import { MenuLanguage } from "@/components/menuLanguage/menuLanguage";
+
 export default function HomePage() {
   const { theme, setTheme } = useTheme();
+  const { language } = React.useContext(LanguageContext);
 
   return (
     <div className="flex flex-col min-h-screen">
       <GoogleAnalytics gaId="G-R5SCDC4C8D" />
       <Header />
       <main className="flex flex-col items-center justify-center flex-grow p-6 bg-gray-100 dark:bg-gray-900 pt-24">
         <h1 className="text-4xl font-bold tracking-tight lg:text-5xl mb-6">
-          The
-          <span className="text-blue-h1 dark:text-blue-400"> newest way </span>
-          to Learn English
+          {language == "english" ? "The" : "A"}
```

Comment:
not sure about if to i18n, what about try to use a lib to handle this?

Reply:
The application has few texts (7 or less) , why use a lib for a small problem? Wouldn't that be adding complexity for nothing? The so-called phrase: "killing an ant with a cannon"? (Honest question)